app/monitor.js

```diff
@@ -15,20 +15,30 @@ var async = require('async'),
                 .digest('hex');
         }
     },
-    sortBuilds = function (newBuilds) {
-        var takeDate = function (build) {
-            return build.isRunning ? build.startedAt : build.finishedAt;
-        };
+    sortBuilds = function (newBuilds, sortOrder) {
+        if(sortOrder == "name") {
+            newBuilds.sort(function (a, b) {
+                if(a.project > b.project) return 1;
+                if(a.project < b.project) return -1;
+ 
+                return 0;
+            });
+        }
+        else {
```

Comment:
Please change the sortOrder name to project. So, we can extend it to other build properties in future.